collect_app/src/main/java/org/odk/collect/android/widgets/GridMultiWidget.java

```diff
@@ -67,10 +66,13 @@
 @SuppressLint("ViewConstructor")
 public class GridMultiWidget extends QuestionWidget implements MultiChoiceWidget {
 
-    // The RGB value for the orange background
+    /* The RGB value for the orange background (obsolete)
```

Comment:
If these color values are unnecessary, I think we can delete them all, what do you think?

Reply:
Removed obsolete declarations from `GridWidget` and `GridMultiWidget`

Thanks @huangyz0918